internal/server/server.go

```diff
@@ -156,6 +171,73 @@ func (a *App) runServer(ctx context.Context) error {
 		return fmt.Errorf("could not create aggregated API server: %w", err)
 	}
 
+	const proxyPrefix = "/proxy.placeholder.suzerain-io.github.io/"
+
+	kubeconfig, err := restclient.InClusterConfig()
+	if err != nil {
+		return fmt.Errorf("could not get in-cluster config: %w", err)
+	}
+
+	kubeTransportConfig, err := kubeconfig.TransportConfig()
+	if err != nil {
+		return fmt.Errorf("could not get in-cluster transport config: %w", err)
+	}
+	kubeTransportConfig.TLS.NextProtos = []string{"http/1.1"}
+
+	kubeRoundTripper, err := transport.New(kubeTransportConfig)
+	if err != nil {
+		return fmt.Errorf("could not get in-cluster transport: %w", err)
+	}
+
+	proxy := httputil.NewSingleHostReverseProxy(&url.URL{Scheme: "https", Host: "kubernetes.default"})
```

Comment:
TODO: load `kubernetes.default` from the default service account mount directory instead of assuming it's this value.